src/int.rs

```diff
@@ -1433,6 +1423,31 @@ impl TryFrom<IVec3> for UVec3 {
     }
 }
 
+// A macro to implement abs for unsigned and signed IVecs
```

Comment:
Small nit: could you make this an actual doc comment i.e. `///`

Reply:
I made them doc comments and ran `cargo fmt -all` now.